backend/requests/hardware.sh

```diff
@@ -0,0 +1,105 @@
+http 'http: //localhost:8000/api/hardware/raspberrypi,2-model-b?daysInterval=1'
+// {
```

Comment:
wrong comment, comments in bash are `#` 
but I think you can just erase the response here, just leave the script and it is fine